### PR TITLE
[UIARCH-710] Bump to TS 4.9.5

### PR DESCRIPTION
**Pull request by chrisbodhi**
https://dataworld.atlassian.net/browse/UIARCH-710

- Remove yarn-related files; stick to npm
- Bump the version of TypeScript
- Upgrade packages for getting tests to run, pass

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,43 +32,45 @@
...
   "dependencies": {
     "@babel/plugin-syntax-dynamic-import": "7.2.0",
-    "babel-plugin-flow-to-typescript": "git+https://github.com/Kiikurage/babel-plugin-flow-to-typescript.git#1375d207e9dd85e9e697cf3d90301171050ca77f",
-    "@babel/traverse": "7.5.5",
+    "@babel/traverse": "^7.26.4",
     "@babel/types": "7.17.0",
+    "babel-plugin-flow-to-typescript": "git+https://github.com/Kiikurage/babel-plugin-flow-to-typescript.git#1375d207e9dd85e9e697cf3d90301171050ca77f",
     "commander": "2.20.0",
-    "lodash": "4.17.15",
+    "lodash": "^4.17.21",
```
Comment (doranxenos) [inline package.json]: Remove the "^"?

File: package.json [hunk 2]
```diff
@@ -32,43 +32,45 @@
...
   "peerDependencies": {
     "@babel/core": "7.5.5",
-    "typescript": "4.2.4"
+    "typescript": "4.9.5"
   },
   "dependencies": {
     "@babel/plugin-syntax-dynamic-import": "7.2.0",
-    "babel-plugin-flow-to-typescript": "git+https://github.com/Kiikurage/babel-plugin-flow-to-typescript.git#1375d207e9dd85e9e697cf3d90301171050ca77f",
-    "@babel/traverse": "7.5.5",
+    "@babel/traverse": "^7.26.4",
```
Comment (doranxenos) [inline package.json]: `^` here too

File: src/commitAll.ts [hunk 3]
```diff
@@ -1,20 +1,20 @@
...
     await git.add(".");
   } catch (e) {
     console.log("error adding");
-    throw new Error(e);
+    throw new Error(e as any);
   }
 
   try {
-    await git.commit(message, undefined, { "-n": true });
+    await git.commit(message, undefined, { "-n": null });
```
Comment (doranxenos) [inline src/commitAll.ts]: why did `true` become `null`?

Comment (chrisbodhi) [reply src/commitAll.ts]: Because reading is hard. :D

**Review by doranxenos — APPROVED** [review]

File: src/commitAll.ts (hunk 3 above)
Comment (katz12) [reply src/commitAll.ts]: so wait, did you change these files manually or were the changes generated somehow?

**Pull request merged**